Vagrantfile

```diff
@@ -17,7 +17,7 @@ Vagrant.configure("2") do |config|
         end
         # Fix Virtualbox Guest Additions installation issue
         # https://github.com/hashicorp/vagrant/issues/12095
-        box.vbguest.installer_options = { allow_kernel_upgrade: true }
+        #box.vbguest.installer_options = { allow_kernel_upgrade: true }
```

Comment:
Any particular reason to comment this out, or just a leftover? Not having *allow_kernel_upgrade* set to *true* is known to cause "vagrant up" to fail when the base box gets old enough. This is because kernel headers for the kernel installed in the base box are no longer available and virtualbox guest addition installation fails. So, I would definitely keep this enabled to avoid problems down the line, even if it happened to work ok right now.